core/src/test/java/kafka/testkit/KafkaClusterTestKit.java

```diff
@@ -191,7 +191,7 @@ private KafkaConfig createNodeConfig(TestKitNode node) {
                     controllerNode.metadataDirectory());
             }
             props.put(SocketServerConfigs.LISTENER_SECURITY_PROTOCOL_MAP_CONFIG,
-                    "EXTERNAL:PLAINTEXT,CONTROLLER:PLAINTEXT");
+                    "EXTERNAL:PLAINTEXT,CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT");
```

Comment:
Do we still need those changes?

Reply:
I will check again.
Thanks for your review.

Reply:
revert, thanks for your remind.